basic/src/test/java/org/jboss/hal/testsuite/test/runtime/SimpleWebserviceEndpointTestCase.java

```diff
@@ -159,10 +161,13 @@ public void callServiceAndGetMetrics() throws Exception {
 
         wsePage.navigateInDeploymentsMenu();
 
-        mbuiNavigation.displayMBuiSubTree("subsystem");
```

Comment:
I suggest to mark `org.jboss.hal.testsuite.mbui.MbuiNavigation.MbuiNavigation(WebDriver)` deprecated.